### ISSUE TITLE
Add a more obvious approve button for MRs

### ISSUE THREAD

**Issue by hawicz** [go-gitea/gitea#31010]
### Feature Description

It is currently quite difficult for people to figure out how to approve a MR.  They need to know to first switch to the "Files changed" tab, then they need to pretend to start a review by clicking on the "Review" button, then realize that the comment is optional, and finally click the "Approve" button.
All of this is decidedly non-obvious to anyone that hasn't been shown it before, and difficult to remember even if you have.

Please add an Approve button on the initial conversation page of a MR.

**Comment by silverwind** [comment]
This UI closely mirrors what GitHub has and the primary color should indicate the primary action for a reviewer. I don't think we should introduce a second "Approve" button.

If you strongly think this should change, please provide a mock up how it would look.

**Comment by hawicz** [comment]
Yes, I definitely agree that blindly approving PRs is to be avoided.
However, many reviews don't happen on the "Files Changes" tab, with reviewers looking at commits individually on the "Commits" tab, or even in a clone.  Even when changes _are_ reviewed there, people note their comments on individual lines, resolve those conversations as each one is addressed, and then end up at a loss at to what to do next - they've just finished their review, so have no reason to click on a button that would seem to start a new review. 

An example of what I’d hope to see is the approach that Gitlab uses, where it shows an “Approve” button on its overview page:
<img width="680" alt="gitlab-approval-button-comparison" src="https://github.com/go-gitea/gitea/assets/763777/f71a24fb-7457-4778-b036-23a929266a5c">

For Gitea, it would be nicer if it showed a similar button next to the “doesn't have enough approvals” message, perhaps like this (don't particularly care whether it's blue, or green, or whatever):
<img width="731" alt="gitea-approve-button-mockup" src="https://github.com/go-gitea/gitea/assets/763777/8357e700-0dfd-4916-be06-16a7d8cc743c">
